### PR TITLE
Add security / firewall warning to readme.

### PR DESCRIPTION
**Pull request by Janmm14**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -9,6 +9,12 @@ Information
 -----------
 BungeeCord is maintained by [SpigotMC](https://www.spigotmc.org/) and has its own [discussion thread](https://www.spigotmc.org/go/bungeecord) with plenty of helpful information and links.
 
+### Security warning
+
+As your minecraft servers have to run without authentication (online-mode=false) for bungeecord to work, this poses a new security risk. Users may connect to your servers directly, under any username they wish to use. The kick "If you wish to use IP forwarding, please enable it in your BungeeCord config as well!" does not protect your spigot servers.
```
Comment (md-5) [inline README.md]: BungeeCord

File: README.md [hunk 2]
```diff
@@ -9,6 +9,12 @@ Information
...
+
+To combat this, you need to restrict access to these servers for example with a firewall. See [firewall guide](https://www.spigotmc.org/wiki/firewall-guide/)
```
Comment (md-5) [inline README.md]: firewall (please see firewall guide).

File: README.md (hunk 1 above)
Comment (Janmm14) [inline README.md]:
```suggestion
As your minecraft servers have to run without authentication (online-mode=false) for BungeeCord to work, this poses a new security risk. Users may connect to your servers directly, under any username they wish to use. The kick "If you wish to use IP forwarding, please enable it in your BungeeCord config as well!" does not protect your spigot servers.
```

File: README.md (hunk 2 above)
Comment (Janmm14) [inline README.md]:
```suggestion
To combat this, you need to restrict access to these servers for example with a firewall (please see [firewall guide](https://www.spigotmc.org/wiki/firewall-guide/)).
```

**Pull request closed without merging**